src/setup.js

```diff
@@ -15,6 +15,29 @@
 goog.module('measurementLibrary.setup');
 
 const {DataLayerHelper} = goog.require('helper');
+const {LogLevel, log} = goog.require('measurementLibrary.logging');
+
+/**
+ * @typedef {function(new:StorageInterface, !Object<string, *>)}
+ */
+let StorageConstructor;
```

Comment:
That's probably true. However, I moved the function to a different file and now it's top-level. The type is also used in the DEFAULT_ variables, which is the reason for using a typedef instead of inlining it